tests/UI/campaigns/functional/BO/03_catalog/01_products/06_bulkActions.js

```diff
@@ -131,6 +131,31 @@ describe('BO - Catalog - Products : Bulk actions products', async () => {
     });
   });
 
+  describe('Bulk duplicate products', async () => {
+    it('should filter products by name', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'filterToBulkDuplicate', baseContext);
+
+      await productsPage.filterProducts(page, 'name', 'TO DELETE');
+
+      const textColumn = await productsPage.getProductNameFromList(page, 1);
+      await expect(textColumn).to.contains('TO DELETE');
```

Comment:
Can't you fetch the number of rows filtered and use it for replacing the `4` in line 155 ?